package/CreateTree.test.js

```diff
@@ -27,19 +42,54 @@ class App extends React.Component {
       </div>
     );
   }
-}
+  
+  let container;
+  let treeGraph;
+  
+  describe('test treeGraph', () => {
+    
+    beforeAll(() => {
+      container = document.createElement('div');
+      document.body.appendChild(container);
+      act(() => {
+      ReactDOM.render(<TicTacToe />, container);
+      console.log("container -", container);
+      // console.log("treeGraph -", JSON.stringify(treeGraph));
```

Comment:
Pls remove these comments and console.log.

Reply:
Removed the comments and console logs